apps/snitch_core/test/tools/user_config_test.exs

```diff
@@ -0,0 +1,35 @@
+defmodule Snitch.Tools.UserConfigTest do
+  use ExUnit.Case, async: false
+  alias Snitch.Tools.UserConfig
+
+  describe "configured properly" do
+    setup do
+      Application.put_env(:snitch_core, :calculators, [TaxCalculator, FlatRateCalculator])
+
+      on_exit(fn ->
+        Application.delete_env(:snitch_core, :calculators)
+      end)
+    end
+
+    test "fetch all calculators" do
```

Comment:
Just a stylistic comment:
This module has nothing to do with calculators. To convey the generality of this module in this test you should put things like `:foo` or `:bar` in the app env instead of `:calculators`.